qiskit_machine_learning/circuit/library/raw_feature_vector.py

```diff
@@ -141,14 +146,10 @@ def feature_dimension(self, feature_dimension: int) -> None:
         if int(num_qubits) != num_qubits:
             raise ValueError("feature_dimension must be a power of 2!")
 
-        if self._num_qubits is None or num_qubits != self._num_qubits:
+        if self.num_qubits is None or num_qubits != self.num_qubits:
```

Comment:
While self._num_qubits could have been None I do not think that is the case for this now which uses the QuantumCircuit getter property where 0 is what is returned for an empty cct.

Reply:
Corrected as well, no need for the additional check for `None`.